### DISCUSSION TITLE
How to specify ignored and watched paths?

### DISCUSSION THREAD

**Discussion by austin-agronick** [reloadware/reloadium#200] [Q&A] [answered]
The [documentation on "Specifying Reloadable Files"](https://reloadium.io/documentation/basic-usage/specifying-reloadable-files) mentions a RELOADIUMPATH variable which does not seem to work when added to my Pycharm Django Run Configuration environment variables section. Is there a way this can be specified in the base config? The pip package seems to only include binaries so we can't even see what BaseConfig is. Is there documentation elsewhere that I'm not seeing? Thanks

**Comment by dkrystki** [comment, marked as answer]
If you use PyCharm plugin, reloadable and ignored paths can be defined in `Project Settings` section.

**Reply by austin-agronick** [reply]
seems `Project Settings` is under the settings for reloadium. there is also project settings within pycharm already so that is not very clear in my opinion. thanks for your help and I hope the documentation can include this info and be more detailed in general as it currently indicates a config file is the only way to change these settings
